src/main/resources/VERSION

```diff
@@ -1 +1 @@
-0.9-SNAPSHOT
+0.10-SNAPSHOT
```

Comment:
No need to increase the version here, v0.9 is not released yet

Reply:
Reverted this change